docs/libcurl/curl_version_info.3

```diff
@@ -149,6 +149,9 @@ libcurl was built with support for TLS-SRP. (Added in 7.21.4)
 .IP CURL_VERSION_NTLM_WB
 libcurl was built with support for NTLM delegation to a winbind helper.
 (Added in 7.22.0)
+.IP CURL_VERSION_HSTS
+libcurl was built with support for HSTS
+(Added in 7.61.0)
```

Comment:
Make it `7.62.0`, which seems the likely candidate since we won't merge new features like this before 7.61.0 and the next release with new features will be 7.62.0...